src/components/Link.tsx

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
+import { Link as GatsbyLink } from 'gatsby';
 import { LinkProps, Link as ExtLink } from 'rebass';
 
 const Link = (props: LinkProps) => {
+  if (props.to) {
+    return <GatsbyLink {...props} />;
```

Comment:
Please fix link style, this shouldn't be underlined

Reply:
Done